myst_nbs/case_studies/wrapping_jax_function.myst.md

```diff
@@ -0,0 +1,765 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.13.7
+kernelspec:
+  display_name: pymc-examples
+  language: python
+  name: pymc-examples
```

Comment:
```suggestion
  name: pymc-examples
substitutions:
  extra_dependencies: jax numpyro
```

substitutions metadata was missing from the myst

Reply:
I guess it doesn't get copied automatically? Or did I forget something